support-frontend/assets/helpers/forms/checkouts.ts

```diff
@@ -341,6 +341,7 @@ function getAvailablePaymentRequestButtonPaymentMethod(
 
 // ----- Exports ----- //
 export {
+	round,
```

Comment:
Why do we need a new export? Can't we use the same rounding function as the old checkout?

Reply:
Hmmm, this has been a thorny problem this afternoon! The old checkout does not multiply the price by 1.04 in the same way its done in the one-time-checkout. Oddly am getting something like `60 x 1.04 = 62.40000000001` hence using the round. Will spend a bit more time looking into a workaround without it now.

Reply:
Ok have used `roundToDecimalPlaces` as used in old checkout, not needing to expose `round`